package.json

```diff
@@ -66,7 +66,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jest": "^24.3.6",
-    "eslint-plugin-jsdoc": "^36.1.0",
+    "eslint-plugin-jsdoc": "^39.2.0",
```

Comment:
Hmm. This generates peer dependency warnings, because it is outside the range accepted by our ESLint config. I understand that this was required for Node.js v18 compatibility though.

I guess we can ignore the peer dependency warning for now if that's easiest. I can update the ESLint config in a separate PR to resolve it.